src/ui/Controllers/CourseController.cs

```diff
@@ -433,7 +433,7 @@ private CourseViewModel LoadViewModel(ProviderSummary org, Domain.Models.Course
 
                         return new SiteViewModel
                         {
-                            ApplicationsAcceptedFrom = courseSite.ApplicationsAcceptedFrom,
+                            ApplicationsAcceptedFrom = courseSite.ApplicationsAcceptedFrom.ToString(),
```

Comment:
If the downward chain expects null than its broken behaviour.

```c#
var s = new SiteViewModel() {
ApplicationsAcceptedFrom = courseSite.ApplicationsAcceptedFrom.ToString(),
}

s.ApplicationsAcceptedFrom
// "" rather null


```


```cs
if(string.IsNullOrEmpty(s.ApplicationsAcceptedFrom))

// instead of 
if(null == s.ApplicationsAcceptedFrom)

```

Its compiler safe but it's not the same anymore as its no longer null, most likely only `*.cshtml`.